resources/views/application/create.blade.php

```diff
@@ -23,6 +23,19 @@
                 app.</p>
         @endif
 
+        @if ($invitingApplication)
+            <div class="alert alert-info text-center">
+                <div class="w-50 mx-auto">
+                    <h3>Confirm Invitation</h3>
+                    <p>
+                        To finalize accepting your invitation from <em>{{ $invitingApplication->getFullName() }}</em>
```

Comment:
Text is a bit confusing to read. Suggestion:

To accept your invitation from ... (...) and become part of their dealership as Share, please review the data below and click on "Submit your application".